stairway/src/test/java/bio/terra/stairway/impl/EnumerateFlightsTest.java

```diff
@@ -150,24 +148,27 @@ public void enumTest() throws Exception {
     assertThat(flightEnum.getTotalFlights(), equalTo(6));
 
     // Case 11: sorting in ascending order
-    filter =
-            new FlightFilter()
-                    .submittedTimeSortDirection(FlightFilterSortDirection.ASC);
+    filter = new FlightFilter().submittedTimeSortDirection(FlightFilterSortDirection.ASC);
     flightList = flightDao.getFlights(0, 100, filter);
     checkResults("case 11", flightList, Arrays.asList("0", "1", "2", "3", "4", "5"));
+    // explicitly verify that classnames are returned as expected (note that the flights list is in
+    // ascending order)
+    assertThat(
+        "class names are correct",
+        flightList.stream().map(FlightState::getClassName).collect(Collectors.toList()),
+        contains(flights.stream().map(FlightState::getClassName).toArray()));
 
     // Case 12: sorting in descending order
-    filter =
-            new FlightFilter()
-                    .submittedTimeSortDirection(FlightFilterSortDirection.DESC);
+    filter = new FlightFilter().submittedTimeSortDirection(FlightFilterSortDirection.DESC);
     flightList = flightDao.getFlights(0, 100, filter);
     checkResults("case 12", flightList, Arrays.asList("5", "4", "3", "2", "1", "0"));
   }
 
   private void checkResults(String name, List<FlightState> resultlList, List<String> expectedIds) {
     List<String> actualIds =
         resultlList.stream().map(FlightState::getFlightId).collect(Collectors.toList());
-    assertThat(name + ": elements match in the correct order", actualIds, contains(expectedIds.toArray()));
+    assertThat(
+        name + ": elements match in the correct order", actualIds, contains(expectedIds.toArray()));
```

Comment:
It looks like there's a common pattern here that could be abstracted, if you wanted to. The abstraction is the getter on the object, so the general method is something like 
```
assertSameField(List<FlightState> expected, List<FlightState> actual, Function<FlightState, T> getter)
```

Reply:
Yeah, maybe...though they are subtly different so it may not be super commonizable